shared/constants/platform.native.js

```diff
@@ -7,12 +7,18 @@ NativeModules.ObjcEngine || {
   appVersionName: 'fallback',
   appVersionCode: 'fallback',
   usingSimulator: 'fallback',
+  isDeviceSecure: 'fallback',
 }
 const isStoryBook = (NativeModules.Storybook && NativeModules.Storybook.isStorybook) || false
 const version = nativeBridge.version
 const appVersionName = nativeBridge.appVersionName
 const appVersionCode = nativeBridge.appVersionCode
 const isSimulator = nativeBridge.usingSimulator === '1'
+// Currently this is given to us as a boolean, but no real documentation on this, so just in case it changes in the future.
+// Android only field that tells us if there is a lock screen.
+const isDeviceSecureAndroid: boolean = typeof nativeBridge.isDeviceSecure === 'boolean'
```

Comment:
We do enforce this in the java side so I don’t think we have to be worried about it that much. This is the sole consumer so I’m not worried about it being out of sync

Reply:
I was worried about the RN side since it's type is just Object.

Reply:
It works through RN. Something like this:
```
 @Override
  public Map<String, Object> getConstants() {
    final Map<String, Object> constants = new HashMap<>();
    constants.put(DURATION_SHORT_KEY, Toast.LENGTH_SHORT);
    constants.put(DURATION_LONG_KEY, Toast.LENGTH_LONG);
    return constants;
  }
```

I was a little paranoid that at some future point RN could convert the values to strings.